walsamlee/politico
# #163416647 Add Create Party Endpoint

#### What does this PR do?
Add `create-party` API endpoint
#### Description of Task to be completed?
Have `/api/v1/parties` endpoint working
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-v1-create-party-163416647`
- Using postman, `POST` with API endpoint URL `/api/v1/parties` 
#### What are the relevant pivotal tracker stories?
#163416647
